### PR TITLE
Api functionalities

### PR DESCRIPTION
**Pull request by Manpreet-hub**
- Added weather API
- Added Quotes API

Screenshot (135)

### CODE REVIEW

File: src/services/getWeatherData.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+export const getWeatherData = async (latitude, longitude, setWeatherInfo) => {
+  const wheatherAPI = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=ff35f8965b5aa598cd3e3e536823bbad&units=metric`;
+
+  try {
+    const getData = await axios.get(wheatherAPI);
+    console.log(getData);
```
Comment (parthgabani777) [inline src/services/getWeatherData.js]: You can remove comment if not necessary

File: src/components/Weather.js [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
+        longitude: position.coords.longitude,
+      });
+    });
+  }, []);
+
+  useEffect(() => {
+    const { latitude, longitude } = location;
+    if (latitude !== "" && longitude !== "")
+      getWeatherData(latitude, longitude, setWeatherInfo);
+    else getWeatherData(28.7041, 77.1025, setWeatherInfo);
```
Comment (parthgabani777) [inline src/components/Weather.js]: You can directly use this value in useState as an initial state

**Review by parthgabani777 — APPROVED** [review]
Folder structure looks well structured and clean.

**Review by ashishnnnnn — APPROVED** [review]
Code seems pretty much clean..
Need to do some changes suggested by parth

File: src/components/Weather.js [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
-    <>
-      <div>
-        <div className="temp text para-md">28 ° </div>
-        <div className="temp-location text">Delhi,India</div>
-      </div>
-    </>
+    <div>
+      <img
+        src={`http://openweathermap.org/img/wn/${weatherInfo.icon}@2x.png`}
+        width="60px"
```
Comment (ArjunGTX) [inline src/components/Weather.js]: Isnt it better to give responsive width and height?

File: src/services/getWeatherData.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+export const getWeatherData = async (latitude, longitude, setWeatherInfo) => {
+  const wheatherAPI = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=ff35f8965b5aa598cd3e3e536823bbad&units=metric`;
```
Comment (ArjunGTX) [inline src/services/getWeatherData.js:4]: API key can be stored as ENV

**Review by ArjunGTX — APPROVED** [review]
Code looks good, have suggested some nickpicks

File: src/services/getWeatherData.js (hunk 1 above)
Comment (Manpreet-hub) [reply src/services/getWeatherData.js]: Done [b4475d4](https://github.com/Manpreet-hub/Browser-extension/pull/2/commits/b4475d420cfd72898055f4bf8b0952982862e7bb)

File: src/components/Weather.js (hunk 3 above)
Comment (Manpreet-hub) [reply src/components/Weather.js]: yes, will work on responsiveness later.

**Pull request merged**